From 16d510d562cd8d2d10fb74880c5da67aede6d279 Mon Sep 17 00:00:00 2001
From: Subrahmanyam Nilla <snilla@marvell.com>
Date: Fri, 8 Mar 2019 14:16:14 +0530
Subject: [PATCH 249/255] soc: octeontx2-dpi: disable DPI PF driver by default
 in Kconfig

commit 973880f2ce797f2b0cc4f94d17fdce55bfeb484d from
git@git.assembla.com:cavium/WindRiver.linux.git

Change-Id: I01ea0e0d5a8fbc464f16a392bfaa468f405b6318
Signed-off-by: Subrahmanyam Nilla <snilla@marvell.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/soc/marvell/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/soc/marvell/Kconfig b/drivers/soc/marvell/Kconfig
index 3821423ee03d..d96cbc842071 100644
--- a/drivers/soc/marvell/Kconfig
+++ b/drivers/soc/marvell/Kconfig
@@ -70,7 +70,6 @@ config OCTEONTX2_RM_DOM_SYSFS
 config OCTEONTX2_DPI_PF
 	tristate "OcteonTX2 DPI-DMA PF driver"
 	depends on ARM64 && PCI
-	default y
 	---help---
 	  Select this option to enable DPI PF driver support.
 	  DPI (DMA packet interface) provides DMA support for MAC.
-- 
2.17.1

